src/status_im2/subs/wallet_2/networks.cljs

```diff
@@ -0,0 +1,33 @@
+(ns status-im2.subs.wallet-2.networks
+  (:require [clojure.string :as string]
+            [re-frame.core :as re-frame]
+            [quo2.foundations.resources :as resources]))
+
+(re-frame/reg-sub
+ :wallet-2/filtered-networks-by-mode
+ :<- [:wallet-2/networks]
+ (fn [[networks] [_ test?]]
+   (let [filter-fn (if test? :Test :Prod)]
+     (map filter-fn
+          networks))))
+
+(def default-network-details
```

Comment:
this info is a bit different than how design displays it so I had to overwrite it for now